src/Form/Theme.php

```diff
@@ -56,6 +67,20 @@ abstract class Theme {
 	abstract public function getOptionsConfig();
 
 
+	/**
+	 * Theme template manager get template according to view.
+	 * Note: Do not forget to call this function before close bracket in overridden getTemplate method
+	 *
+	 * @param string $template
+	 *
+	 * @return string
+	 * @since 2.7.0
+	 */
+	public function getTemplate( $template ) {
```

Comment:
We should be consistent here. Either change `$this->views` to `$this->templates`, or change `getTemplate` to `getView`.

Reply:
fixed: https://github.com/impress-org/givewp/compare/667bf05cc9e7...8584bd4f57e9